.github/workflows/phpunit.yml

```diff
@@ -26,6 +26,11 @@ on:
         description: PHP versions to test, separated by comma.
         required: true
         type: string
+      phpunitConfig:
+        description: PHPUnit configuration file.
+        default: phpunit.xml.dist
```

Comment:
By default should be empty.

From [docs](https://phpunit.readthedocs.io/en/9.5/organizing-tests.html?highlight=phpunit.xml.dist#composing-a-test-suite-using-xml-configuration):

> If phpunit.xml or phpunit.xml.dist (in that order) exist in the current working directory and --configuration is not used, the configuration will be automatically read from that file.